### PR TITLE
widget tableview: add new function lv_tabview_set_tab_name() to change a tab name during runtime

### PR DESCRIPTION
**Pull request by mibcat**
Hi !
I have a similar [use case](https://forum.lvgl.io/t/how-to-change-the-tab-name-in-a-tabview-and-update-the-screen-with-the-new-name/1254) and want to update the tab name dynamically during runtime.
But looking into `lv_tabview.c` code I realized that this could not be just a one-liner :smile: 

Because I'm not very familiar with the **lvgl** framework in detail there might be room for improvements ...

### CODE REVIEW

File: src/lv_widgets/lv_tabview.c [hunk 1]
```diff
@@ -448,6 +448,36 @@ void lv_tabview_set_tab_act(lv_obj_t * tabview, uint16_t id, lv_anim_enable_t an
...
+ * Set the name of a tab.
+ * @param tabview pointer to Tab view object
+ * @param id index of a tab to load
+ * @param name new tab name
+ */
+void lv_tabview_set_tab_name(lv_obj_t *tabview, uint16_t id, char *name)
+{
+  LV_ASSERT_OBJ(tabview, LV_OBJX_NAME);
+
+  // get tabview's ext pointer which contains the tab name pointer list
```
Comment (embeddedt) [inline src/lv_widgets/lv_tabview.c]: C-style comments, please. `/* */` :slightly_smiling_face:

**Review by embeddedt — CHANGES_REQUESTED** [review]

File: src/lv_widgets/lv_tabview.c (hunk 1 above)
Comment (mibcat) [reply src/lv_widgets/lv_tabview.c]: done

**Review by embeddedt — APPROVED** [review]

**Comment by kisvegabor** [comment]
Looks good. Please add a few lines about it in the docs.
See: https://docs.lvgl.io/latest/en/html/contributing/index.html#improve-the-docs

**Comment by mibcat** [comment]
Cloning the lvgl/docs repo from my fork and switching to dev branch gives me the following (so far unseen) error message:
```bash
> git checkout dev
fatal: 'dev' could be both a local file and a tracking branch.
Please use -- (and optionally --no-guess) to disambiguate
```

But I can't see anything unusual:
```bash
> git remote show origin 
* remote origin
  Fetch URL: git@github.com:mibcat/docs.git
  Push  URL: git@github.com:mibcat/docs.git
  HEAD branch: master
  Remote branches:
    dev        tracked
    latest     tracked
    master     tracked
    release/v6 tracked
    release/v7 tracked
  Local branch configured for 'git pull':
    master merges with remote master
  Local ref configured for 'git push':
    master pushes to master (up to date)
```

therefore I worked on with:
`git checkout origin/dev` followed by a `git checkout dev` again

Hope this had no negative effects ... PR is on the way

**Comment by kisvegabor** [comment]
It's because dev is also a directory.

`git checkout dev --` should work.

**Comment by kisvegabor** [comment]
I merged the PR, thank you.

One last thing: please add a line to CHANGELOG.md.

**Comment by kisvegabor** [comment]
Great! Thank you very much! :slightly_smiling_face:

**Comment by mibcat** [comment]
added to V7.2.0 - hope this was the correct location (because branch dev is mentioned in V7.3.0)

**Comment by kisvegabor** [comment]
Ups, `v7.2` is in `master` and `v7.3` in `dev`. So it should be under `v7.3`. I've updated it here: https://github.com/lvgl/lvgl/commit/7e38ac68e3e2a84fabbb70b5462ba2b78b01284f

**Comment by mibcat** [comment]
Ok - thanks for correction.

**Pull request merged**